From 8b4cb9d332b84db970bf446ad9576ad98ba448e5 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 14 Jan 2012 21:45:18 -0500
Subject: [PATCH] build: Add --disable-documentation option

This may sound weird for gnome-doc-utils, but basically I need a
way to bootstrap an operating system without documentation initially,
because it's the source of big dependency cycles.

This configure option is enough to get the m4 macros installed, and
then move from there to building GDM without documentation too.
---
 configure.in       |    5 +++++
 xml2po/Makefile.am |    6 +++++-
 2 files changed, 10 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 15d97a4..dd56f21 100644
--- a/configure.in
+++ b/configure.in
@@ -21,6 +21,11 @@ AC_PROG_LN_S
 gdu_cv_have_gdu=yes
 GNOME_DOC_DEFINES
 
+AC_ARG_ENABLE([documentation],
+	AS_HELP_STRING([--enable-documentation], [enable documentation generated via XSLT]),
+	[], [enable_documentation=yes])
+AM_CONDITIONAL(ENABLE_DOCUMENTATION, test x$enable_documentation = xyes)
+
 AC_ARG_ENABLE([build-utils],
 	AC_HELP_STRING([--enable-build-utils],
 	[build and install the build utilities [default=yes]]),,
diff --git a/xml2po/Makefile.am b/xml2po/Makefile.am
index 2f17f0b..4aff003 100644
--- a/xml2po/Makefile.am
+++ b/xml2po/Makefile.am
@@ -3,7 +3,11 @@ SUBDIRS = xml2po examples
 pkgconfigdir = $(datadir)/pkgconfig
 pkgconfig_DATA = xml2po.pc
 
-man_MANS = xml2po.1
+man_MANS = 
+
+if ENABLE_DOCUMENTATION
+man_MANS += xml2po.1
+endif
 
 $(srcdir)/README: README.in
 	sed -e "s/\@VERSION\@/@VERSION@/g" < README.in > README
-- 
1.7.6.4

